src/main/resources/com/google/api/codegen/go/sample.snip

```diff
@@ -155,12 +164,42 @@
   }
 @end
 
-@private callChain(methodNameComponents)
-  @join resource : methodNameComponents on "."
+@private field(field)
+  @if field.description
+    @join line : util.getDocLines(field.description, 100)
+      // {@line}
+    @end
+
+
+  @end
+  {@field.name} := {@field.defaultValue}  // TODO: Update placeholder value.
+  @if field.example
+    // {@field.example}
+  @end
+@end
+
+@private callChain(class)
+  {@resourceCallChain(class)}({@paramList(class.methodCallFieldVarNames)}){@optionalFieldCallChain(class)}
+@end
+
+@private resourceCallChain(class)
```

Comment:
This is not really a call; maybe just `methodResource`? Might make sense for the `{@class.ServiceVarName}` to go here.